tests/integration-tests/tests/cloudwatch_logging/test_cloudwatch_logging.py

```diff
@@ -179,7 +179,7 @@ def _get_initial_cluster_log_state(self):
                 continue
             elif tags.get("Name", "") == "HeadNode":
                 self._set_head_node_instance(instance)
-            else:
+            elif self.scheduler == "slurm":  # AWS Batch Compute instances do not use CloudWatch
```

Comment:
shall we have a `!= awsbatch` condition so that if we add new schedulers we assume by default that they support CloudWatch?

Reply:
Done